src/plone/restapi/tests/http-examples/inherit_expansion.resp

```diff
@@ -0,0 +1,83 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "@components": {
+        "actions": {
+            "@id": "http://localhost:55001/plone/document/@actions"
+        },
+        "aliases": {
+            "@id": "http://localhost:55001/plone/document/@aliases"
+        },
+        "breadcrumbs": {
+            "@id": "http://localhost:55001/plone/document/@breadcrumbs"
+        },
+        "contextnavigation": {
+            "@id": "http://localhost:55001/plone/document/@contextnavigation"
+        },
+        "navigation": {
+            "@id": "http://localhost:55001/plone/document/@navigation"
+        },
+        "navroot": {
+            "@id": "http://localhost:55001/plone/document/@navroot"
+        },
+        "plone.navigationroot": {
```

Comment:
@davisagli  For the expansion shouldn't we also group the behaviours inside inherit field? Like for navigation,breadcrumbs..?

Reply:
@nileshgulia1 I was trying to keep it flatter, but you're right, it will be more consistent with the other expanders and avoid the risk that a behavior name collides with an expander name. Done in https://github.com/plone/plone.restapi/pull/1887/commits/c572fadbbd3077217eec50e4279fc64435035cc8